v19.1/managed-build-a-python-app.md

```diff
@@ -0,0 +1,225 @@
+
+1. In the top right corner of the Console, click the **Connect** button.
+
+    The **Connect** modal displays.
+
+    <img src="{{ 'images/v19.1/managed/connect-modal.png' | relative_url }}" alt="Connect to cluster" style="border:1px solid #eee;max-width:100%" />
+
+3. From the **User** dropdown, select `maxroach`.
+4. Select a **Region** to connect to.
+5. From the **Database** dropdown, select `defaultdb`.
+6. On the **Connect from Shell** tab, click **Copy connection string**.
+
+    This is how you will access the built-in SQL client later. You will need to replace the `<certs_dir>` placeholders with the path to your `certs` directory.
+
+7. Click the **Download ca.crt** button.
+8. Create a `certs` directory on your local workstation and move the `ca.crt` file to the `certs` directory.
+
+## Step 4: Create the Managed CockroachDB database
+
+On the machine where you want to run the CockroachDB SQL client:
+
+1. [Download the CockroachDB binary](install-cockroachdb.html):
+
+    <div class="filters clearfix">
+      <button style="width: 15%" class="filter-button" data-scope="mac">Mac</button>
+      <button style="width: 15%" class="filter-button" data-scope="linux">Linux</button>
+    </div>
+    <p></p>
+
+    <section class="filter-content" markdown="1" data-scope="mac">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ curl https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.darwin-10.9-amd64.tgz \
+    | tar -xJ
+    ~~~
+    </section>
+
+    <section class="filter-content" markdown="1" data-scope="linux">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ wget -qO- https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.linux-amd64.tgz \
+    | tar  xvz
+    ~~~
+    </section>
+
+2. Copy the binary into the `PATH` so it's easy to run the SQL client from any location:
+
+    <section class="filter-content" markdown="1" data-scope="mac">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cp -i cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach /usr/local/bin
+    ~~~
+    </section>
+
+    <section class="filter-content" markdown="1" data-scope="linux">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo cp -i cockroach-{{ page.release_info.version }}.linux-amd64/cockroach /usr/local/bin
+    ~~~
+    </section>
+
+3. Use the connection string generated in Step 3 to connect to CockroachDB's built-in SQL client:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach sql --url 'postgres://maxroach@<region>.<cluster_name>:26257/defaultdb?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    ~~~
+
+2. Create a database `todos`:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > create database todos;
+    ~~~
+
+3. Use database `todos`:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > USE todos;
+    ~~~
+
+3. Create a table `todos`:
```

Comment:
This works (and it's simpler than introducing another tool), but most users will want to use some sort of schema migration tool like `alembic` or `sqlalchemy-migrate`. We may want to build this into a future version of the example app and then move the table creation down below so it uses the application connection string.